src/status_im2/contexts/quo_preview/preview.cljs

```diff
@@ -278,15 +278,18 @@
 (defn customization-color-option
   ([]
    (customization-color-option {}))
-  ([{:keys [feng-shui?] :as opts}]
-   (merge {:key     :customization-color
-           :type    :select
-           :options (->> (merge colors/customization (when feng-shui? {:feng-shui nil}))
-                         keys
-                         sort
-                         (map (fn [k]
-                                {:key k :value (string/capitalize (name k))})))}
-          opts)))
+  ([{:keys [feng-shui? set-type] :as opts}]
```

Comment:
nit: `set` here is slightly confusing because of "set", "get" etc. Maybe something else will help avoid that. color-type? color-set also very clear imo 👍

Reply:
I deleted it completely since it wasn't needed anymore